ktxspec.adoc

```diff
@@ -535,7 +535,7 @@ If `faceCount` is equal to 6, `pixelHeight` must be equal to
 `pixelWidth`, and `pixelDepth` must be 0.
 
 `pixelHeight` must not be 0 for block-compressed formats, including
-BasisLZ/ETC1S and UASTC.
+BasisLZ/ETC1S, UASTC, UASTC HDR and GPU Photo
```

Comment:
Missing dot at the end.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/1f207fa65dd60a700ee88e6758d1ed67b2a8e577.